extension/observer/dockerobserver/config.go

```diff
@@ -71,14 +55,12 @@ func (config Config) Validate() error {
 func (config *Config) Unmarshal(conf *confmap.Conf) error {
 	err := conf.Unmarshal(config)
 	if err != nil {
-		if floatAPIVersion, ok := conf.Get("api_version").(float64); ok {
-			return fmt.Errorf(
-				"%w.\n\nHint: You may want to wrap the 'api_version' value in quotes (api_version: \"%1.2f\")",
-				err,
-				floatAPIVersion,
-			)
-		}
 		return err
 	}
-	return nil
+
+	if len(config.ExcludedImages) == 0 {
```

Comment:
I am not certain this is strictly needed, zero and empty lists are effectively the same when iterating overthem.